doc/articles/get-started-wizard.md

```diff
@@ -2,19 +2,68 @@
 uid: Uno.GetStarted.Wizard
 ---
 
-# Welcome to Uno Platform
+<!-- markdownlint-disable MD026 -->
+# Done! Important next steps:
 
-**Congratulations, you've just created a new project using the [Uno Platform](https://platform.uno/) project templates!**
+Create a simple cross-platform counter application.  
+Choose one of the 4 counter variants combining markup language (XAML or C# Markup) and presentation framework (MVVM or MVUX).
```

Comment:
Maybe capitalize the work `counter` or say "Counter app variants"?

Just seeing "counter variants" makes it sound like something else

Reply:
Agreed with Steve. 

Also, lets please add a newline after this sentence. for better readability  "Create a simple cross-platform counter application."

And also lets capitalize C in Counter in that sentence please: "Create a simple cross-platform Counter application."